packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx

```diff
@@ -31,3 +47,290 @@ export const Uncontrolled = () => {
+    </IcTextField>
+  </div>
+);
+
+export const MultiLineTextAreaWithResize = (): ReactElement => (
+  <div style={style}>
+    <IcTextField
+      rows={6}
+      resize
+      label="What is your favourite coffee?"
+      placeholder="Please enter…"
+      helperText="Such as Arabica, Robusta or Liberica"
+    ></IcTextField>
+  </div>
+);
+
+export const TextFieldWithValidation = (): ReactElement => (
+  <>
+    <div style={style}>
+      <IcTextField
+        maxLength={25}
+        value="Arabica"
+        label="What is your favourite coffee?"
+        required
+        placeholder="Please enter…"
+        helperText="Such as Arabica, Robusta or Liberica"
+        validationStatus="success"
+        validationText="Good choice!"
+      ></IcTextField>
+    </div>
+    <div style={style}>
+      <IcTextField
+        maxLength={25}
+        value="Arabica"
+        label="What is your favourite coffee?"
+        required
+        placeholder="Please enter…"
+        helperText="Such as Arabica, Robusta or Liberica"
+        validationStatus="warning"
+        validationText="There’s only a few left! Order now to avoid missing out."
+      ></IcTextField>
+    </div>
+    <div style={style}>
+      <IcTextField
+        maxLength={25}
+        value="Arabica"
+        label="What is your favourite coffee?"
+        required
+        placeholder="Please enter…"
+        helperText="Such as Arabica, Robusta or Liberica"
+        validationStatus="error"
+        validationText="Now it has really gone to (coffee) pot"
+      ></IcTextField>
+    </div>
+  </>
+);
+
+export const TextFieldWithInlineValidation = (): ReactElement => (
+  <div style={style}>
+    <IcTextField
+      maxLength={25}
+      value="Arabica"
+      label="What is your favourite coffee?"
+      required
+      placeholder="Please enter…"
+      helperText="Such as Arabica, Robusta or Liberica"
+      validationStatus="success"
+      validationText="Good choice!"
+      validationInline
+    ></IcTextField>
+  </div>
+);
+
+export const ReadonlyTextField = (): ReactElement => (
+  <div style={style}>
+    <IcTextField
+      maxLength={25}
+      value="Arabica"
+      label="What is your favourite coffee?"
+      placeholder="Please enter…"
+      helperText="Such as Arabica, Robusta or Liberica"
+      validationStatus="success"
+      validationText="Good choice!"
+      readonly
+    ></IcTextField>
+  </div>
+);
+
+export const CustomSizeTextField = (): ReactElement => (
+  <div style={style}>
+    <IcTextField
+      style={{ "--input-width": "40%" }}
+      label="What is your favourite coffee?"
+      value="Arabica"
+      max-length="25"
+      required
+      placeholder="Placeholder"
+      helper-text="Such as Arabica, Robusta or Liberica"
+    >
+      <SlottedSVG
+        slot="icon"
+        xmlns="http://www.w3.org/2000/svg"
+        height="24px"
+        viewBox="0 0 24 24"
+        width="24px"
+        fill="#000000"
+        size="small"
```

Comment:
does this `size` on slottedsvg do anything? i don't think there is a "size" attribute. can this be removed & then just use the ReusableSlottedIcon?

Reply:
As agreed, removed the icon from CustomSizeTextField